### ISSUE TITLE
Popup: closed while "mouse click inside but release outside" of the Popup block

### ISSUE THREAD

**Issue by jiayimu** [Semantic-Org/Semantic-UI-React#3554]
### Steps
I am creating a login popup component in React by using React Semantic UI Popup. I am using "Events triggering the popup on click". I was trying to copy the content in the Popup block, but the popup was closed when my mouse released outside of the popup box.  

```
import React, { Component } from 'react';
import { Button, Message, Popup, Icon, Menu } from 'semantic-ui-react';

// code inside of render return function
<Popup trigger={<Button icon>LOGIN</Button>}
       on='click'
       className='login-popup'
>
       <div className='popup-main'>
              <Message attached='bottom'>
                        Log In
              </Message>
              <LoginForm
                  {...this.props}
              />
        </div>

        <Message attached='bottom'>
              Don't have account? Signup instead.
        </Message>
</Popup>
```

### Expected Result
I would like to keep the Popup open unless the user clicks the mouse outside of the Popup

### Actual Result
The Popup closed when the mouse clicked inside and released outside of the popup box.

### Version
0.85.0

### Testcase
I tested it on the offical website and still get the same result.

**Comment by layershifter** [comment]
https://react.semantic-ui.com/modules/popup/#usage-click

I tried this example and everything works as expected. Please provide a minimal CodeSandbox example and detailed steps.

**Comment by jiayimu** [comment]
@layershifter 
Thank you for your checking. I think I might not explain the issue clearly. I asked this in stack overflow ([here](https://stackoverflow.com/questions/55502216/react-semantic-ui-popup-was-hidden-while-mouse-click-inside-but-release-outside)). I updated my expected result as below. Hope the update makes more sense.

**Expected Result**:  
The Popup box should not be closed if the mousedown event happens inside of the Popup, no matter where the mouseup event triggered.

**Comment by layershifter** [comment]
Fixed in #3575.

**Comment by mstarcevic** [comment]
@jiayimu 

I came across your post while trying to resolve a similar issue. I eventually resolved it by setting one of the input fields in the popup to autoFocus. Prior to that, clicking on an icon or somewhere not specifically an input, closed the popup straight away.

I see the issue has been closed but thought the comment might be helpful in any case.
